src/plugins/vis_builder/public/application/utils/use/use_saved_vis_builder_vis.ts

```diff
@@ -74,6 +74,13 @@ export const useSavedVisBuilderVis = (visualizationIdFromUrl: string | undefined
           data.query.filterManager.setAppFilters(actualFilters);
           data.query.queryString.setQuery(query);
 
+          chrome.recentlyAccessed.add(
```

Comment:
Seems the call does not comply with the interface of `chrome.recentlyAccessed.add`.